### DISCUSSION TITLE
Speed Dial update

### DISCUSSION THREAD

**Discussion by yonasuriv** [dailydotdev/daily#619] [Feature Requests]
With the daily.dev sidebar, speed dial is going on background, and being such a necessary feature but with little customization.

It shows only 8 favorites but they become 4 when you use 

- **browsers with sidebars** like Vivaldi or Opera, for example
- or when **using smaller devices**.

![3](https://user-images.githubusercontent.com/59540565/176865738-8528c1cd-1d13-401f-b4b6-867cd4657bba.jpg)

You have to **set the zoom to 90%**, also the **news tiles changes from rows of 3 to 2**, making them a bit smaller would help also.
![4](https://user-images.githubusercontent.com/59540565/176865754-c77e55c9-5cfa-4881-b4a4-c1a8aee8597b.jpg)

An option to set the speed dial on the center, or add a bit more than 8 would be awesome, since the daily.dev app itself it's the **best one** by far

Starting with something simple, like from this:
![1](https://user-images.githubusercontent.com/59540565/176865069-5c96f51e-6b86-4c4a-ade7-8c249044428e.jpg)

To this:
![2](https://user-images.githubusercontent.com/59540565/176865188-43679292-0b8e-48cf-a020-9fa28f097855.jpg)

.

**Comment by idoshamun** [comment]
Hi @yonasuriv,

Let me forward it to the team and see what can be done. I'll keep you posted

**Reply by idoshamun** [reply]
@yonasuriv we will adjust the smaller resolution to have more icons :)

**Reply by yonasuriv** [reply]
You guys are the best by far
